nexus/proposals/tests/test_models.py

```diff
@@ -0,0 +1,43 @@
+# Third Party Stuff
+from django.test import TestCase
+from tests import factories as f
+
+# nexus Stuff
+from nexus.proposals.models import Proposal, ProposalKind
+
+
+class TestProposalModel(TestCase):
+
+    def setUp(self):
+        self.user = f.create_user(email='test@example.com', password='test')
+        self.proposalkind = f.create_proposal_kind(kind='talk')
```

Comment:
Please follow python naming conventions.

This should be `proposal_kind`

Reply:
I'm sorry for the mistake. I have fixed it.